content/millennium/r4/financial/general/insurance-plan.md

```diff
@@ -36,11 +44,18 @@ Search for InsurancePlans that meet supplied query parameters:
 
 ### Parameters
 
-Name                  | Required?          | Type          | Description
-----------------------|--------------------|---------------|-------------------------------------------------------
-`_id`                 | This or `owned-by` | [`token`]     | The logical resource id associated with the resource.
-`owned-by`            | Yes                | [`reference`] | Organization associated with the InsurancePlan.
+Name       | Required      | Type          | Description
+-----------|---------------|---------------|-------------------------------------------------------
+`_id`      | Conditionally | [`token`]     | The logical resource ID associated with the resource. This parameter is required if the `owned-by` parameter is not used. Example: `12345`
+`owned-by` | Conditionally | [`reference`] | The organization associated with the insurance plan. This parameter is required if the `_id` parameter is not used. Example: `12345`
+
+_Implementation Notes_
 
+- The `_id` and `owned-by` search parameters cannot be provided together.
+- When searching with the `_id` parameter:
+  - It can be provided with either a single reference, or a comma-separated list of references. Example: `_id=12345` or `_id=12345,67890`
```

Comment:
do we need to state this here? since it applies to all resources (as far as I know..)

Reply:
I've seen some resources that like this one which will accept a list, and some that return an error message that multiples don't work.

Would you prefer no detail there?